### ISSUE TITLE
https://sketch2code.azurewebsites.net/ not working

### ISSUE THREAD

**Issue by Joanne201** [microsoft/ailab#164]
I do not know what is wrong but whenever i try to go in https://[sketch2code.azurewebsites.net](https://sketch2code.azurewebsites.net/)/ it just say I cant reach the page. If you have any solution please tell me.

**Comment by lfrigodesouza** [comment]
Yeah, I really wanted to try it, but it looks like it's unavailable for some time now.

**Comment by Otto404** [comment]
On 12/12/23 the site is still down
